src/workbench/SideBar.tsx

```diff
@@ -143,7 +133,13 @@ const SideBarContents = ({ panes, ...props }: SideBarContentsProps) => {
       >
         <TabList>
           {/* Top margin aims to align with other logo. */}
-          <Box width="3.75rem" mt="1.2rem" ml="auto" mr="auto" mb="11.5vh">
+          <Box
+            width="3.75rem"
+            mt="1.2rem"
+            ml="auto"
+            mr="auto"
+            mb="max(11.5vh, 7.7rem)"
```

Comment:
This change prevents the curve of the first tab overlapping with the coloured area at the top of the toolkit. Will revisit when we get search designs.